src/libs/antares/memory/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+set(PROJ memory)
+set(HEADERS
+        include/antares/${PROJ}/memory.h
+        include/antares/${PROJ}/memory.hxx
+        include/antares/${PROJ}/new_check.hxx
+)
+set(SRC_MEMORY
+        ${HEADERS}
+        memory.cpp)
+source_group("memory" FILES ${SRC_MEMORY})
+
+add_library(${PROJ} ${SRC_MEMORY})
+add_library(Antares::${PROJ} ALIAS ${PROJ})
+
+target_link_libraries(${PROJ}
+        PRIVATE
+        yuni-static-core
+        logs
```

Comment:
Headers from yuni and logs appear in public headers of this lib
--> it means that we must make those dependencies PUBLIC, not PRIVATE, otherwise transitive dependency inclusion will not work.